### ISSUE TITLE
RWR problematic

### ISSUE THREAD

**Issue by bwanaaa** [b04rdr1d3r/F5E-Streamdeck-profile-for-DCS-Interface#1]
I cannot get the RWR buttons to respond properly. Have you?

**Comment by b04rdr1d3r** [comment]
Hi, part of the issue comes from the fact they are only partly mapped. some RWR buttons are three states, when DCS for streamdeck only has room for two states. that means that we would need to duplicate buttons to have all the lights function properly.... tbh, the F-5E being low priority atm, I did not spend time exploring further and only mapped the most basic functionalities....

**Comment by bwanaaa** [comment]
Thank you for taking the time to answer. Nice icons btw. Did you make them?On Feb 8, 2023, at 12:47 PM, Blue Storm ***@***.***> wrote:﻿
Hi, part of the issue comes from the fact they are only partly mapped. some RWR buttons are three states, when DCS for streamdeck only has room for two states. that means that we would need to duplicate buttons to have all the lights function properly.... tbh, the F-5E being low priority atm, I did not spend time exploring further and only mapped the most basic functionalities....

**Comment by bwanaaa** [comment]
Yes. Thats how i made the fuel transfer switch- with two buttons. On Feb 8, 2023, at 12:47 PM, Blue Storm ***@***.***> wrote:﻿
Hi, part of the issue comes from the fact they are only partly mapped. some RWR buttons are three states, when DCS for streamdeck only has room for two states. that means that we would need to duplicate buttons to have all the lights function properly.... tbh, the F-5E being low priority atm, I did not spend time exploring further and only mapped the most basic functionalities....
